app/server/routes/v1/comment/comment.js

```diff
@@ -84,6 +84,8 @@ router.post("/", async (req, res) => {
       creator: user.id,
       post: post.id,
       community: post.community,
+      "upvotes.count": 0,
+      "downvotes.count": 0,
       createdOn: moment().format("MMMM Do YYYY, h:mm:ss a"),
```

Comment:
We might want to follow the date standard of largest to smallest here i.e., YYYY MM DD.

Reply:
This is how the date looks `June 5th 2022, 5:34:02 pm`, I think you're think of the format for when its a date like 2022/03/11. Do you think it should use the number format or written format? In terms of readability I think the way it is is better, but please share your thoughts